aidial_adapter_bedrock/llm/model/claude/v3/tokenizer.py

```diff
@@ -192,6 +192,11 @@ def _tokenize_tool_system_message(
             ChatCompletionDeployment.ANTHROPIC_CLAUDE_V3_HAIKU
             | ChatCompletionDeployment.ANTHROPIC_CLAUDE_V3_HAIKU_US
             | ChatCompletionDeployment.ANTHROPIC_CLAUDE_V3_HAIKU_EU
+            # Actually token usage for Haiku 3.5 is unknown
+            # temporary using tha same as for Haiku 3
+            # will be not actual after Anthropic tokenizer integration
```

Comment:
The last line is redundant. 
We already have an issue for this: https://github.com/epam/ai-dial-adapter-bedrock/issues/171, no need to repeat.